src/main/java/com/icthh/xm/ms/configuration/repository/impl/MemoryConfigStorageImpl.java

```diff
@@ -394,4 +405,12 @@ private List<Configuration> getByPaths(Collection<String> paths, Function<Config
         return result;
     }
 
+    private List<Configuration> getByAntPatternPaths(Collection<String> antPatternPaths) {
```

Comment:
suggest returning Stream to avoid double transformation (stream --> to list -- to stream)